Expecto/Expecto.fs

```diff
@@ -879,7 +879,8 @@ module Tests =
       |> List.length
 
     if count > 0 then
-      config.printer.info "It was requested that no focused tests exist, but yet there are %d focused tests found."
+      sprintf "It was requested that no focused tests exist, but yet there are %d focused tests found." count
```

Comment:
Could we possibly do it with the templating from Logary? `...but there are {count} focused tests found" >> setField "count" count)`?